src/main/java/microsoft/exchange/webservices/data/ComplexPropertyCollection.java

```diff
@@ -427,8 +427,7 @@ public int getCount() {
   public TComplexProperty getPropertyAtIndex(int index)
       throws IllegalArgumentException {
     if (index < 0 || index >= this.getCount()) {
-      throw new IllegalArgumentException("parameter \'index\' : " +
-          Strings.IndexIsOutOfRange);
+      throw new IllegalArgumentException("parameter \'index\' : " + "index is out of range.");
```

Comment:
I think this is up to embed the current index into the message but this will result in a message like where no index is printed. The escaped `'` also seem useless.